source/ai-data-usage.txt

```diff
@@ -0,0 +1,36 @@
+.. vscode-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+The MongoDB Github Copilot Participant is powered by Generative AI (Gen
+AI), and may give inaccurate responses. See our `Generative AI FAQ
+<https://dochub.mongodb.org/core/faq-ai-features>`__ for more
+information about Gen AI in MongoDB products.
+
+Third Party Providers
+---------------------
+
+The MongoDB Github Copilot Participant is powered by `Github Copilot <https://github.com/features/copilot>`__.
+This is subject to change in the future. 
```

Comment:
I'm not sure if this will ever change since it is an extension for Github Copilot.

Reply:
Removed the line about changing!